build.sbt

```diff
@@ -1,57 +1,30 @@
-import ReleaseTransformations._
 import xerial.sbt.Sonatype._
 
-organization := "de.gccc.sbt"
 scalaVersion := "2.12.15"
-sonatypeProfileName := "de.gccc"
-publishMavenStyle := true
-licenses := Seq("APL2" -> url("http://www.apache.org/licenses/LICENSE-2.0.txt"))
-
-sonatypeProjectHosting := Some(GitHubHosting("schmitch", "sbt-jib", "c.schmitt@briefdomain.de"))
-homepage := Some(url("https://github.com/schmitch"))
-scmInfo := Some(
-  ScmInfo(
-    url("https://github.com/schmitch/sbt-jib"),
-    "scm:git@github.com:schmitch/sbt-jib.git"
-  )
-)
-developers := List(
-  Developer(
-    id = "schmitch",
-    name = "Christian Schmitt",
-    email = "c.schmitt@briefdomain.de",
-    url = url("https://github.com/schmitch")
-  )
-)
```

Comment:
I moved them to `inThisBuild` clause as that is how it is done in the `scala-ci-release`. Despite me using scala for quite some time I still don't get difference between these two approaches. Better be safe and follow instructions :)